ams/server/src/main/java/com/netease/arctic/server/optimizing/plan/AbstractPartitionPlan.java

```diff
@@ -51,12 +50,10 @@ public abstract class AbstractPartitionPlan implements PartitionEvaluator {
   private long toSequence = INVALID_SEQUENCE;
   protected final long planTime;
 
-  protected final Map<IcebergDataFile, List<IcebergContentFile<?>>> fragmentFiles = Maps.newHashMap();
-  protected final Map<IcebergDataFile, List<IcebergContentFile<?>>> segmentFiles = Maps.newHashMap();
-  protected final Map<String, Set<IcebergDataFile>> equalityDeleteFileMap = Maps.newHashMap();
-  protected final Map<String, Set<IcebergDataFile>> posDeleteFileMap = Maps.newHashMap();
-  
-  private List<SplitTask> splitTasks;
+  protected final Map<IcebergDataFile, List<IcebergContentFile<?>>> rewriteDataFiles = Maps.newHashMap();
+  protected final Map<IcebergDataFile, List<IcebergContentFile<?>>> rewritePosDataFiles = Maps.newHashMap();
+  // protected Delete files are Delete files related to Data files not optimized in this plan
+  protected final Set<String> protectedDeleteFiles = Sets.newHashSet();
```

Comment:
The name `protectedDeleteFiles` is somewhat ambiguous. Would `keepedDeleteFiles` or `reservedDeleteFiles` be more appropriate?

Reply:
OK, I prefer `reservedDeleteFiles`.

Reply:
LGTM.